events/system/interactionCreate.js

```diff
@@ -29,14 +30,20 @@ export default {
         });
       }
 
+      interaction.dbUser = dbUser;
+      interaction.activeSession = await sessionService.getOneActive(dbUser.id);
```

Comment:
Would it not be better to have a set place for all this like 
```suggestion
      const session = await sessionService.getOneActiveByDiscordId(interaction.user.id);
      interaction.context = {
        user: dbUser, // or just user if above changed.
        session,
        settings,
        isProDuck,
        // any other existential baggage your user might have
      };
```
The when we grab in the commands its
```js
const { user, session, settings, isProDuck } = interaction.context;
```
It gives us a separation from the discord stuff and its easier to console log context for a bigger picture when problems arise.